src/extension/monitor/core/src/main/java/org/geoserver/monitor/auditlog/AuditLogger.java

```diff
@@ -76,8 +73,6 @@
 
     public AuditLogger(MonitorConfig config, GeoServerResourceLoader loader) throws IOException {
         this.config = config;
-        Resource monitoring = loader.get("monitoring");        
-        defaultPath = monitoring.dir().getAbsolutePath();
```

Comment:
So no need to checkout dir() contents here - we are just going after the single file below.